extension/global_files/browserDetect.js

```diff
@@ -6,67 +6,90 @@
  *
  * @returns {string}
  */
-var browserDetect = function() {
```

Comment:
Maybe something to look into the future, but maybe there's a npm module that can be imported to do this functionality? However, no action needed in this PR. 

My original reasoning for storing it into redux cache is so that the background and options scripts don't have to recompute the browser detect. But now I'm thinking of ways of reducing complexity for the future.